pkgs/development/libraries/kde-frameworks/extra-cmake-modules/default.nix

```diff
@@ -19,11 +20,22 @@ mkDerivation {
 
   outputs = [ "out" ]; # this package has no runtime components
 
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [
+    bash
+  ];
+
+  # note: these will be propagated into the same list extra-cmake-modules is in
   propagatedBuildInputs = [
```

Comment:
I am undecided if this is correct or propagatedNativeBuildInputs

Reply:
It is correct by my understanding: `extra-cmake-modules` in nativeBuildInputs => propagatedBuildInputs are also put in nativeBuildInputs. See #64992

In fact anything you put in propagatedNativeBuildInputs would be ignored.

Reply:
Oh yeah, I forgot this is wrong, lol. Those should probably be saturating, not overflowing... But whatever, out of scope.